bluebird-promisell.js

```diff
@@ -0,0 +1,49 @@
+/* jshint esnext:true */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const Promise = require('bluebird');
+const P = require('bluebird-promisell');
+const R = require('ramda');
+const S = require('sanctuary');
+
+//    readFile :: Object -> String -> Promise Error String
+const readFile = S.flip(Promise.promisify(fs.readFile));
+
+//    readFileWithDirAndName :: String -> String -> Promise Error String
+const readFileWithDirAndName = R.curryN(2, (dir, name) => {
+  const filePath = path.join(dir, name);
+  return readFile({ encoding: 'utf8' })(filePath);
+});
```

Comment:
I was thinking that if `readFile` takes `encoding` as an argument, so should `readFileWithDirAndName`. Conversely, if `readFileWithDirAndName` does *not* take `encoding` as an argument, neither should `readFile`. It seems to me these two functions should be consistent in this respect unless there's a reason for them not to be.